src/js/components/stateLanding/table/StateLandingTable.jsx

```diff
@@ -23,13 +23,19 @@ const propTypes = {
 const StateLandingTable = (props) => {
     const hideBody = props.loading || props.error || props.results.length === 0 ? 'state-list__body_hide' : '';
 
-    const body = props.results.map((row) => (
+    // Check if the search string is a known state abbreviation
```

Comment:
We are already searching on the `code` (abbreviation) here: https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/containers/stateLanding/StateLandingContainer.jsx#L108, so when the code is added to `name` in the model (see comment below) it should automatically be highlighted.